docs/guide/input.md

```diff
@@ -113,9 +113,16 @@ The app splits the screen in to quarters, with a `RichLog` widget in each quarte
 
 You can move focus by pressing the ++tab++ key to focus the next widget. Pressing ++shift+tab++ moves the focus in the opposite direction.
 
+### Focusable widgets
+
+Each widget has a boolean `can_focus` attribute which determines if it is capable of receiving focus.
```

Comment:
Could we also document that you can set `can_focus=True` with the subclass args?

Reply:
Where would you like me to do that? At this point in the guide we haven't described how to create even a basic widget.

Reply:
I guess it would have to be in the [widgets](https://textual.textualize.io/guide/widgets/) section of the guide.

It could use a short example as well.

Reply:
That section of the docs is feeling quite bare - it only shows how to create static widgets that display some text. There's no mention of widget-level bindings at all and so no interactivity (other than the Rich markup links). In fact, I don't think there's a single example of bindings on a widget in the entire guide.

Can I suggest we add an example of a simple interactive widget with `can_focus` demonstrated too? Otherwise I think the `can_focus=True` example would look out of place?

Reply:
Yup. Makes sense. Can you run it by me when you have a rough draft please.

Reply:
Added a section